datareduction/datareductionoperator.cpp

```diff
@@ -1484,13 +1614,7 @@ namespace DRO {
 	       if (ENERGY > E1limit) thread_E1_sum += block_data[n * SIZE_VELBLOCK+cellIndex(i,j,k)] * ENERGY * DV3;
 	       if (ENERGY > E2limit) thread_E2_sum += block_data[n * SIZE_VELBLOCK+cellIndex(i,j,k)] * ENERGY * DV3;
             }
-         }
```

Comment:
What do these changes have to do with the sauerkraut? That looks wrong to me, isn't it?

Reply:
(this line and the block below)

Reply:
indeed - a merge mixup?

Reply:
Yes, l. 1489-1493 somehow disappeared in the merge with Markus's branch, and I am to blame for deleting l. 1487 to fix the resulting error...
I am adding those lines back.

Reply:
Yes, l. 1489-1493 somehow disappeared in the merge with Markus's branch, and I am to blame for deleting l. 1487 to fix the resulting error...
I am adding those lines back.